jersey-servers/src/main/java/com/palantir/remoting3/servers/jersey/RemoteExceptionMapper.java

```diff
@@ -40,35 +39,25 @@
  * response is then thrown as a {@link RemoteException} at the call point in server A.  If server A does not catch this
  * exception and it raises up the call stack back into Jersey, execution enters this {@link RemoteExceptionMapper}.
  * <p>
- * To preserve debuggability, the exception and HTTP status code from B's exception are logged at WARN level and
- * returned back to the caller/browser.
+ * To preserve debuggability, the exception and HTTP status code from B's exception are logged at WARN level, but not
+ * propagated to caller to avoid an unintentional dependency on the remote exception.
  */
 @Provider
 final class RemoteExceptionMapper implements ExceptionMapper<RemoteException> {
 
     private static final Logger log = LoggerFactory.getLogger(RemoteExceptionMapper.class);
+    private static final ServiceExceptionMapper mapper = new ServiceExceptionMapper();
 
     @Override
     public Response toResponse(RemoteException exception) {
         Status status = Status.fromStatusCode(exception.getStatus());
+        ServiceException servieException = new ServiceException(ErrorType.INTERNAL, exception);
```

Comment:
WOuld prefer to inline the relevant bits of ServiceException mapper to avoid getting another stacktrace